packages/x-components/src/design-system-deprecated/components/filter/default.scss

```diff
@@ -1,4 +1,4 @@
-@import '../../mixins';
+@import '../../../design-system/mixins';
 
 .x-filter {
```

Comment:
Since the `x-components` are not using `x-filter` class anymore, what do you think of renaming the deprecated styles as well to `x-facet-filter`? This way the deprecated styles will be properly applied when used along with the `x-components` filters that, from now on, use the new nomenclature. 

The same would happen with the `x-filter--is-selected` to `x-selected` renaming.